addons/mail/controllers/thread.py

```diff
@@ -14,6 +14,14 @@ def mail_thread_data(self, thread_model, thread_id, request_list):
         thread = request.env[thread_model].with_context(active_test=False).search([("id", "=", thread_id)])
         return thread._get_mail_thread_data(request_list)
 
+    @http.route("/mail/thread/followers", methods=["POST"], type="json", auth="user")
```

Comment:
What is the route adding compared to directly calling a model method ? Seems it mainly adds complexity, as in the end everything depends on user rights, then do a search and a format.